react-effects/my-app/src/List.tsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable @typescript-eslint/no-unused-vars -- Remove me */
+import { useEffect, useState } from 'react';
+import { readItems, type Item } from './read';
+
+export function List() {
+  const [isLoading, setIsLoading] = useState(true);
+  const [items, setItems] = useState<Item[]>([]);
+  const [error, setError] = useState<unknown>();
+
+  // Your code here:
+  //  - When the component mounts:
+  //    - Read the items using `readItems` and update state so the list displays
+  //    - Handle errors from `readItems`
+  useEffect(() => {
+    async function read() {
+      try {
+        const response = await readItems();
+        setItems(response);
+        setIsLoading(false);
+      } catch (error) {
+        setError(error);
```

Comment:
You also need to set isLoading to false in this branch or you will never see the error message.